File: src/main/java/net/slipp/domain/user/PasswordDto.java
package net.slipp.domain.user;

public class PasswordDto {

    private Long id;
    private String oldPassword;
    private String newPassword;
    private String newPasswordConfirm;
    
    public PasswordDto() {
    }
    
    public PasswordDto(Long id) {
        this.id = id;
    }

    public PasswordDto(Long id, String oldPassword, String newPassword, String newPasswordConfirm) {
        this.id = id;
        this.oldPassword = oldPassword;
        this.newPassword = newPassword;
        this.newPasswordConfirm = newPasswordConfirm;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public void setOldPassword(String oldPassword) {
        this.oldPassword = oldPassword;
    }

    public void setNewPassword(String newPassword) {
        this.newPassword = newPassword;
    }

    public void setNewPasswordConfirm(String newPasswordConfirm) {
        this.newPasswordConfirm = newPasswordConfirm;
    }

    public String getOldPassword() {
        return oldPassword;
    }

    public String getNewPassword() {
        return newPassword;
    }

    public String getNewPasswordConfirm() {
        return newPasswordConfirm;
    }
}

File: src/main/scala/net/slipp/domain/user/PasswordDto.scala
package net.slipp.domain.user

import java.lang.Long
import scala.beans.BeanProperty

class PasswordDto(i: Long, oPass: String, nPass: String, nPasswordConfirm: String) {
  @BeanProperty
  var id: Long = i

  @BeanProperty
  var oldPassword: String = oPass

  @BeanProperty
  var newPassword: String = nPass

  @BeanProperty
  var newPasswordConfirm: String = nPasswordConfirm

  def this() = this(null, null, null, null)

  def this(id: Long) = this(id, null, null, null)
}
